CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Coming soon
-- remove several deprecated APIs
+
+## [0.9.0] - 2020-10-23
+- Remove several deprecated APIs.
```

Comment:
I don't think we actually removed these yet.  Let's keep it in the "coming soon".